lib/main.dart

```diff
@@ -1,67 +1,315 @@
+import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
-
+import 'package:flutter/services.dart';
 void main() {
-  runApp(const MyApp());
+  WidgetsFlutterBinding.ensureInitialized();
+  SystemChrome.setPreferredOrientations([DeviceOrientation.landscapeLeft])
+      .then((_) {
+  runApp(MyApp());
+  });
 }
 
+var marginDivider = 1;
+double orientationWidth = 1;
+double orientationHeight = 1;
+
 class MyApp extends StatelessWidget {
```

Comment:
Your `main.dart` is kind of big (300+). While it's fine for the lab, usually at this mark you should start thinking about decomposing your `HomePage` to multiple smaller widgets and, potentially, putting them into different files (but not mandatory)